docs/src/styles/pages/docs.css

```
@@ -0,0 +1,191 @@
+  position: relative;
+  overflow: visible;
+}
+
+section .title {
+  margin: 1em 0 0 0;
+}
+
+pre,
+code {
+  border-radius: var(--rounded-sm);
+  corner-shape: squircle;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)